gwlfe/CalcCnErosRunoffSed.py

```diff
@@ -163,8 +163,8 @@ def CalcCN(z, i, Y, j):
             if z.Water >= 0.2 * z.CNumPervReten:
                 z.QrunP[l] = (z.Water - 0.2 * z.CNumPervReten) ** 2 / (z.Water + 0.8 * z.CNumPervReten)
 
-        # TODO: Should 11 be NRur + 1? What is this trying to do?
-        lu = l - 11
+        # TODO: What is this trying to do?
+        lu = l - 10
```

Comment:
I think your code is correct. 

`ISRR` and `ISRA` should have a length of `6` (which is `NUrb`, https://github.com/WikiWatershed/gwlf-e/blob/develop/gwlfe/parser.py#L995). And this loop iterates from `NRur` to `NLU` which should be 10 to 16